client/my-sites/email/email-management/home/email-plan-warnings.jsx

```diff
@@ -56,45 +57,57 @@ class EmailPlanWarnings extends React.Component {
 		);
 	}
 
-	renderWarningCTA( warning ) {
-		const warningSlug = warning.warning_slug;
-		if ( ! warningSlug ) {
-			return null;
-		}
+	renderCTAForGooglePendingTOSAcceptance() {
+		const { domain, translate } = this.props;
+
+		return (
+			<Button compact primary href={ getGoogleAdminUrl( domain.name ) } target="_blank">
+				{ translate( 'Finish Setup' ) }
+				<Gridicon icon="external" />
+			</Button>
+		);
+	}
 
-		if ( EMAIL_WARNING_SLUG_UNUSED_MAILBOXES === warningSlug ) {
-			if ( EMAIL_ACCOUNT_TYPE_TITAN_MAIL === this.props.emailAccountType ) {
-				return this.renderWarningCTAForTitanUnusedMailboxes();
+	renderCTA() {
+		const { emailAccount } = this.props;
+
+		if ( hasUnusedMailboxWarnings( emailAccount ) ) {
+			if ( isTitanMailAccountType( emailAccount ) ) {
```

Comment:
Would this not be cleaner using `&&` rather than a nested `if`?

Reply:
While that seems more readable, I preferred **this** approach then because it seemed that a certain point in the future we would implement similar behaviour for Google Workspace.

However I think we can revert when that gets implemented.